osu.Game/Tests/Visual/OsuTestScene.cs

```diff
@@ -46,13 +46,27 @@ public abstract class OsuTestScene : TestScene
         protected Storage LocalStorage => localStorage.Value;
 
         private readonly Lazy<DatabaseContextFactory> contextFactory;
+
+        protected IAPIProvider API
+        {
+            get
```

Comment:
this can be shortened to `get => dummyAPI ?? throw new Exception(...)` since `dummyAPI` is created only if `UseOnlineAPI` is false

Reply:
i think it reads better as-is.